fhir-path/src/main/java/com/ibm/fhir/path/evaluator/FHIRPathEvaluator.java

```diff
@@ -1262,22 +1369,58 @@ private EvaluationContext(FHIRPathTree tree) {
             this.tree = tree;
         }
         
+        /**
+         * Get the FHIRPath tree associated with this EvaluationContext
+         * 
+         * @return
+         *     the FHIRPath tree associated with this EvaluationContext
+         */
         public FHIRPathTree getTree() {
             return tree;
         }
         
+        /**
+         * Set an external constant using a name and FHIRPath node
+         * 
+         * @param name
+         *     the name
+         * @param node
+         *     the FHIRPath node
+         */
         public void setExternalConstant(String name, FHIRPathNode node) {
             externalConstantMap.put(name, singleton(node));
         }
         
+        /**
+         * Set an external constant using a name and a collection of FHIRPath node
```

Comment:
how is it set?  to each node in the collection?

Reply:
The collection is the value of the constant. If the constant is a single value, then the collection would be a singleton.